common/src/chain/transaction/signature/inputsig/arbitrary_message/mod.rs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) 2021-2024 RBB S.r.l
+// opensource@mintlayer.org
+// SPDX-License-Identifier: MIT
+// Licensed under the MIT License;
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://github.com/mintlayer/mintlayer-core/blob/master/LICENSE
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+const MESSAGE_MAGIC_PREFIX: &str = "===MINTLAYER MESSAGE BEGIN===";
+const MESSAGE_MAGIC_SUFFIX: &str = "===MINTLAYER MESSAGE END===";
```

Comment:
Does it make sense to put some newlines around it, so the prefix/suffix are on separate lines when printed?

Reply:
Yeah maybe, but it doesn't matter all that much. These are just added as protection from spoofing transaction signatures challenges.